recognition/include/pcl/recognition/impl/linemod/line_rgbd.hpp

```diff
@@ -87,7 +74,7 @@ template <typename PointXYZT, typename PointRGBT> bool
 pcl::LineRGBD<PointXYZT, PointRGBT>::loadTemplates (const std::string &file_name, const size_t object_id)
 {
   // Open the file
-  int ltm_fd = pcl_open (file_name.c_str (), O_RDONLY | O_CLOEXEC);
```

Comment:
This extra flag was `O_CLOEXEC` was added in #2755, supposedly by clang-tidy and now removed again. Why?